src/flight/sixaxis.c

```diff
@@ -171,24 +178,29 @@ static void sixaxis_wait_for_still() {
 
     now = time_micros();
     last_data = data;
+    ++loop_counter;
   }
+  return loop_counter < 20;
 }
 
 void sixaxis_gyro_cal() {
-  float limit[3];
```

Comment:
I can't catch it. 
I thought the gyro calibration is used to get the gyro bias.  i.e. the reading of gyro when it stays still. nothing related to the gyro orientation ( it reads near 0 (angular speed) no matter what orientation it is)
i think it has a limit because it shouldn't be a very large value. (there must be a move if it's large).
and the wrong bias will cause the quad to rotate on arm.

BTW, `limit` was already removed in this commit. https://github.com/BossHobby/QUICKSILVER/commit/9f56ee79a611307c10b9cf390253f240b9114555
I just removed the left code of limit, whichi has acturally no side effect.

Reply:
oh gosh, you are absolutely right and i was thinking about accel calibration.
sorry, i reviewed that on my phone. that makes this discussion mute.